### ISSUE TITLE
Unable to get the focus on "Hide all" of the main "Toggle visibility"  icon

### ISSUE THREAD

**Issue by bkutty** [fgpv-vpgf/fgpv-vpgf#1515]
Tested Build :http://fgpv.cloudapp.net/demo/v1.5.0-2/prod/samples/index-fgp-en.html?keys=JOSM,EcoAction,CESI_Other,NPRI_CO,Railways,mb_colour,Airports,Barley

While testing for keyboard accessibility, I was unable to access the "hide all" option of the visibility icon.  It didn't seem to be an issue for the "show All" option. Exiting from this option using "Esc" also didnt work. Please investigate

![image](https://cloud.githubusercontent.com/assets/18539315/21191971/e01ecf74-c1f4-11e6-8a0c-9b6b0148e9bd.png)

**Comment by AleksueiR** [comment]
Focus manager problem most likely: the first item is disabled and cannot get focus. Focus manager will try to stick focus somewhere though instead of letting it go to the beginning of the page, so it might be one farther up the focus history. From the screenshot, the focus seems to be on the "JOSM" legend entry.

**Comment by jolevesq** [comment]
@AleksueiR To prevent this should we set the show all and hide all always enable?

**Comment by AleksueiR** [comment]
We can, but it create an option that sometimes doesn't do anything (trying to hide all the layers when they are already hidden). Ideally, I would like to avoid that. 

This issue seems related to the focus manager setting focus on newly opened menus and dialogs. @milesap, as the focus manager mastermind, do you have any thoughts on that?

**Comment by milesap** [comment]
Quick note after speaking with @bkutty: focus manager is only supported in Chrome. Since it works relatively well in FF and IE, I have let it remain active there as well. If we notice it degrades the overall experience in FF or IE we can consider selectively deactivating FM. 

I believe FM is Microsoft Edge compatible, so when that rolls around and I have the time, I'll include that in the supported category.
